tests/conftest.py

```diff
@@ -35,5 +35,12 @@ def load_export_data(path: Path) -> dict:
 def load_mapping(path: Path) -> dict:
     data = load_export_data(path)
     if os.environ.get("ENV") == "dev":
-        data["credential"] = {**data["credential"], "host": "localhost", "port": 15432}
+        data["credential"] = {
+            **data["credential"],
+            "host": "localhost",
+            "port": 5432,
+            "login": "river",
+            "password": "river",
+            "database": "river",
```

Comment:
where does this come from ? in the docker-compose.test.yml file, login/password/db are `test` and port is `15432`